web/pages/home.jsx

```diff
@@ -0,0 +1,122 @@
+export function Home() {
+  return (
+    <>
+      <div className="container-fluid cover min-vh-100 p-0">
+        <div className="row justify-content-center m-0">
+          <div className="col-12 col-md-6 text-light d-flex justify-content-end align-items-center py-5">
+            <p>
+              <span className="fs-2">Ready to level up your device?</span>
+              <br />
+              <span className="fs-2 fw-bold">
+                LETS BUILD YOUR DREAM PC <br />
+                TOGETHER!
+              </span>
+            </p>
+          </div>
+          <div className="col-12 col-md-6 d-flex">
+            <img
+              src="/img/home/custom_pc-s.png"
+              className="m-auto w-75"
+              alt=""
+            />
+          </div>
+        </div>
+      </div>
+      <div className="container py-3">
+        <div className="fs-2 fw-medium my-3">Browse By Category</div>
+        <div className="p-3 d-flex flex-row overflow-x-auto text-center">
+          <a
+            href="/v1/products"
```

Comment:
can you change all the links in this page by removing the `/v1`
It will result in a 404 but that's okay. The `/products` page is being worked on in parallel

```suggestion
            href="/products"
```

Reply:
Oh Sure